db_stress_tool/db_stress_common.cc

```diff
@@ -13,6 +13,7 @@
 #include <cmath>
 
 ROCKSDB_NAMESPACE::DbStressEnvWrapper* db_stress_env = nullptr;
+ROCKSDB_NAMESPACE::DbStressEnvWrapper* trace_env = nullptr;
```

Comment:
I can add a comment here.  The main purpose of having a different env is because db_stress_env might enable fault_injection, while we don't want the trace_env to have any fault-injection.

In other words, if the fault-injection is off, then trace_env will be identical to db_stress_env.